### PR TITLE
SK-1516-WEB-Email-configure_mail_apple

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
FR update of Mail on macOS

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 1]
```diff
@@ -30,11 +39,40 @@ Les comptes MX Plan peuvent être configurés sur différents logiciels de messa
...
+> **Etape 2**
+>> Saisissez, dans la fenêtre « **Ajouter un compte Mail** », les informations suivantes : <br><br>- Un **Nom** pour votre compte e-mail <br>- Votre **Adresse e-mail** <br>- Le **Mot de passe** de votre adresse e-mail<br>
+>> ![mailmac](images/mail-mac-email02.png){.thumbnail .w-400 .h-600}
+>>
+> **Etape 3**
+>> Dans la fenêtre suivante, complétez les informations :<br>
+>> - **Adresse e-mail** 
+>> - **Nom d'utilisateur** : Saisissez votre adresse e-mail complète 
+>> - **Mot de passe**
+>> - **Type de compte** : Sélectionnez `POP` ou `IMAP` (recommandé)
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
>> - **Type de compte** : Sélectionnez `IMAP` (recommandé) ou `POP`
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 2]
```diff
@@ -30,11 +39,40 @@ Les comptes MX Plan peuvent être configurés sur différents logiciels de messa
...
+>>
+>> > [!warning]
+>> >
+>> > Il est normal de voir apparaître le message en rouge « **impossible de vérifier le nom ou le mot de passe du compte** » lorsque la fenêtre apparaît la première fois. Néanmoins, si ce message persiste après validation, cela signifie que les informations saisies sont erronées.<br><br>
+>>
+>> ![mailmac](images/mail-mac-email03.png){.thumbnail .w-400 .h-600}
+
+> [!warning]
+>
+> Si, après avoir suivi les étapes de configuration ci-dessous, vous rencontrez un défaut d'envoi ou de réception. Consultez la rubrique [Modifier les paramètres existants](#modify-settings)
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
> Si, après avoir suivi les étapes de configuration ci-dessous, vous rencontrez un défaut d'envoi ou de réception, consultez la rubrique [Modifier les paramètres existants](#modify-settings)
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 3]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
 
 Si votre compte e-mail est déjà paramétré et que vous devez accéder aux paramètres du compte pour les modifier :
 
 - Cliquez sur `Mail`{.action} dans la barre de menu en haut de votre écran, puis sur `Préférences`{.action}.
-- Sélectionez le compte concerné dans la colonne de gauche puis cliquez sur `Réglages du serveur`{.action}.
+- Sélectionnez le compte concerné dans la colonne de gauche puis cliquez sur `Réglages du serveur`{.action}.
+- Dans la section `Serveur de réception (POP)` ou `Serveur de réception (IMAP)`, saisissez votre adresse e-mail complète dans la case `Nom d'utilisateur`{.action} ainsi que le `Mot de passe`{.action} associé dans la case prévue à cet effet.
+- Dans la section `Serveur de d'envoi (SMTP)` saisissez votre adresse e-mail complète dans la case `Nom d'utilisateur`{.action} ainsi que le `Mot de passe`{.action} associé dans la case prévue à cet effet.
+- Décochez les cases `Gérer automatiquement les réglages de connexion`{.action} pour faire apparaitre les paramètres de `Port`{.action} et `Authentification`{.action}.
+- Assurez-vous que les cases `Utiliser TLS/SSL`{.action} soient cochées.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
- Assurez-vous que les cases `Utiliser TLS/SSL`{.action} sont cochées.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 4]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
+- Dans les menus déroulants `Authentification`{.action}, assurez-vous que `Mot de passe` soit sélectionné.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
- Dans les menus déroulants `Authentification`{.action}, vérifiez que `Mot de passe` est sélectionné.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 5]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
+- Pour les cases `Nom de l'hôte`{.action} et `Port`{.action}, référez-vous aux valeurs de la rubrique « [Rappel des paramètres POP, IMAP et SMTP](#popimap-settings) », **en s'assurant de bien correspondre au type de serveur (IMAP, POP et SMTP) et à votre région (Europe ou Asie-Pacifique)**.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
- Pour les cases `Nom de l'hôte`{.action} et `Port`{.action}, référez-vous aux valeurs de la rubrique « [Rappel des paramètres POP, IMAP et SMTP](#popimap-settings) ». **Vérifiez bien que le type de serveur (IMAP, POP et SMTP) correspond à votre région (Europe ou Asie-Pacifique)**.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 6]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
+
+Pour finaliser la configuration, cliquez sur `Enregistrer`{.action}.
+
+![mailmac](images/mail-mac-email04.png){.thumbnail .w-400 .h-600}
+
+> [!primary]
+>
+> **Changer sa configuration**
+>
+> Si votre adresse e-mail est configurée en **IMAP** et que vous souhaitez changer cette configuration en **POP**, vous devez supprimer le compte sur Mail de MacOS puis le recréer en **POP**, et inversement lorsque votre configuration est initialement en **POP**.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
> Si votre adresse e-mail est configurée en **IMAP** et que vous souhaitez changer cette configuration en **POP**, vous devez supprimer le compte sur Mail de MacOS puis le recréer en **POP**. 
>
> Inversement, lorsque votre configuration est initialement en **POP** et que vous souhaitez changer cette configuration en **IMAP**, vous devez supprimer le compte sur Mail de MacOS puis le recréer en **IMAP**.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 7]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
+|---|---|
+|Nom d'utilisateur|Renseignez l'adresse e-mail **complète**|
+|Mot de passe|Renseignez le mot de passe de l'adresse e-mail|
+|Serveur **EUROPE** (entrant)|imap.mail.ovh.net **ou** ssl0.ovh.net|
+|Serveur **AMERIQUE / ASIE-PACIFIQUE** (entrant)|imap.mail.ovh.ca|
+|Port|993|
+|Type de sécurité|SSL/TLS|
 
-Dans le cadre d'une configuration en **POP**, les valeurs sont les suivantes:
+Pour l'envoi des e-mails, si vous devez renseigner manuellement les paramètres **SMTP** dans les préférences du compte, vous trouverez ci-dessous les paramètres à utiliser :
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
Pour l'envoi des e-mails, si vous devez renseigner manuellement les paramètres **SMTP** dans les préférences du compte, retrouvezci-dessous les paramètres à utiliser :
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 8]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
 > [!primary]
 >
 > **Changer sa configuration**
 >
 > Si votre adresse e-mail est configurée en **IMAP** et que vous souhaitez changer cette configuration en **POP**, vous devez supprimer le compte sur Mail de MacOS puis le recréer en **POP**.
->
+
+### Que faire si je ne parviens pas à recevoir/envoyer mes e-mails ?
+
+- Si vous constatez l'apparition de l'icône visible sur la capture ci-dessous, il s'agit d'une déconnexion réseau. Vérifier que votre connexion internet fonctionne correctement.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
- Si vous constatez l'apparition de l'icône visible sur la capture ci-dessous, il s'agit d'une déconnexion réseau. Vérifiez que votre connexion à Internet fonctionne correctement.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 9]
```diff
@@ -46,40 +84,105 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
+
+![mailmac](images/mail-mac-disconnect.png){.thumbnail .w-400 .h-600}
+
+- Si vous constatez l'apparition de l'icône visible sur la capture ci-dessous, il s'agit d'un défaut de synchronisation. Vérifiez les paramètres de configuration de votre compte e-mail en consultant la rubrique [Modifier les paramètres existants](#modify-settings)
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
- Si vous constatez l'apparition de l'icône visible sur la capture ci-dessous, il s'agit d'un défaut de synchronisation. Vérifiez les paramètres de configuration de votre compte e-mail en consultant la rubrique [Modifier les paramètres existants](#modify-settings).
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md [hunk 10]
```diff
@@ -46,40 +84,104 @@ OVHcloud propose aussi une application web permettant d'accéder à votre adress
...
+|---|---|
+|Nom d'utilisateur|Renseignez l'adresse e-mail **complète**|
+|Mot de passe|Renseignez le mot de passe de l'adresse e-mail|
+|Serveur **EUROPE** (entrant)|imap.mail.ovh.net **ou** ssl0.ovh.net|
+|Serveur **AMERIQUE / ASIE-PACIFIQUE** (entrant)|imap.mail.ovh.ca|
+|Port|993|
+|Type de sécurité|SSL/TLS|
 
-Dans le cadre d'une configuration en **POP**, les valeurs sont les suivantes:
+Pour l'envoi des e-mails, si vous devez renseigner manuellement les paramètres **SMTP** dans les préférences du compte, retrouverez ci-dessous les paramètres à utiliser :
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_mail_macos/guide.fr-fr.md]:
```suggestion
Pour l'envoi des e-mails, si vous devez renseigner manuellement les paramètres **SMTP** dans les préférences du compte, retrouvez ci-dessous les paramètres à utiliser :
```

**Pull request merged**